project-base/templates/Styleguide/styleguide.html.twig

```diff
@@ -319,7 +319,7 @@
                         <a href="#buttons" class="btn">Primary button</a> &nbsp;
                         <a href="#buttons" class="btn btn--success">Success button</a> &nbsp;
                         <a href="#buttons" class="btn btn--success"><i class="svg svg-arrow"></i>Go to cart</a> &nbsp;
-                        <a href="#buttons" class="window-popup__actions__btn--continue window-button-continue btn btn--success">Go to cart <i class="svg svg-arrow"></i></a> &nbsp;
```

Comment:
I would keep both js- and test- class here as it's used for javascript functionality too

Reply:
280701bd083251504b1574975ad579b617fb4f82